src/WalkingModule/src/Module.cpp

```diff
@@ -124,6 +139,36 @@ bool WalkingModule::setRobotModel(const yarp::os::Searchable& rf)
 
 bool WalkingModule::configure(yarp::os::ResourceFinder& rf)
 {
+
+        m_pushRecoveryActiveIndex=0;
+        m_useStepAdaptation = rf.check("use_step_adaptation", yarp::os::Value(false)).asBool();
+        // TODO REMOVE ME
```

Comment:
Leftover?

Reply:
Done!

Reply:
Was it a leftover or there was something to be removed?

Reply:
I forgot to remove that text and I removed that. I think the other parts are ok!